cms/djangoapps/contentstore/views/block.py

```diff
@@ -300,6 +300,118 @@ def xblock_view_handler(request, usage_key_string, view_name):
         return HttpResponse(status=406)
 
 
+@require_http_methods("GET")
+@login_required
+def partial_edit_view_xblock(request, usage_key_string):
```

Comment:
I came out that the biggest part of this function was dedicated to experiments, and isn't used anymore, can you please try to clean up the function content according to comments?

Also, I'd like to change the title (from partial to just edit_view) and docstring, but it could be done before opening upstream PR, thanks!